### PR TITLE
Rafa

### PR DESCRIPTION
**Pull request by rafaelva-z**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -7,28 +7,26 @@
...
     url_for,
     session,
-    flash
+    flash,
+    jsonify
 )
 import pandas as pd
 import numpy as np
 import requests
-from dotenv import load_dotenv
```
Comment (rajh-phuyal) [inline app.py:15]: Is it not gonna be used?

**Review by oks-erm — APPROVED** [review]
noice

File: plots.py
Comment (rajh-phuyal) [inline plots.py:1]: Very clean file. Well done.

**Review by rajh-phuyal — APPROVED** [review]
Nice work. The visuals looks good.

**Pull request merged**